app/services/integrations/aggregator/accounts_service.rb

```diff
@@ -3,20 +3,38 @@
 module Integrations
   module Aggregator
     class AccountsService < BaseService
+      LIMIT = 450
+      MAX_SUBSEQUENT_REQUESTS = 15
```

Comment:
I'm not sure what an "account" is in this context. I'm assuming that it's very unexpected to have so many?